### PR TITLE
Has been added compatibility with PHP 8 to `magento/semver`

### PR DESCRIPTION
**Pull request by andrewbess**
# Description (*)

This pull request (PR) provides compatibility current project with PHP 8.

## Fixed Issues (if relevant)

1. Fixes magento/magento2#32872

## Contribution checklist (*)

- [ ] Pull request has a meaningful description of its purpose
- [ ] All commits are accompanied by meaningful commit messages
- [ ] All new or changed code is covered with unit/integration tests (if applicable)
- [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by ihor-sviziev — APPROVED** [review]

**Review by ihor-sviziev — APPROVED** [review]

**Review by fascinosum — CHANGES_REQUESTED** [review]

**Review by fascinosum — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -8,25 +8,28 @@
     ],
     "bin": ["bin/svc"],
     "require": {
-        "php": "~7.3.0||~7.4.0",
+        "php": "~7.3.0||~7.4.0||~8.0.0",
         "ext-json": "*",
-        "phpstan/phpdoc-parser": "^0.3.5",
+        "laminas/laminas-stdlib": "~3.5.0",
```
[the remark this answers is not in the document]
Comment (ihor-sviziev) [reply composer.json:14]: Could you explain why it's needed? All Zend classes were replaced with Laminas.

[the remark this answers is not in the document]
Comment (ihor-sviziev) [reply composer.json:14]: @fascinosum Could you explain why it's needed? All Zend classes were replaced with Laminas.

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (fascinosum) [reply composer.json:14]: @ihor-sviziev, this plugin will help people who use this library in their CI/CD, to install Laminas packages, even if they have declared dependency on ZF

**Comment by ihor-sviziev** [comment]
@fascinosum, when we can expect a new version for this svc checker?

**Comment by fascinosum** [comment]
hi @ihor-sviziev, I think it will be released when full testing cycle for compatibility with Magento with PHP 8.0 is completed

**Pull request merged**